piker/ui/_graphics/_lines.py

```diff
@@ -682,27 +682,27 @@ def order_line(
         llabel.show()
 
     else:
-        # left side label
-        llabel = line.add_label(
-            side='left',
-            fmt_str=' {exec_type}-{order_type}:\n ${$value}',
-        )
-        llabel.fields = {
-            'order_type': order_type,
-            'level': level,
-            '$value': lambda f: f['level'] * f['size'],
-            'size': size,
-            'exec_type': exec_type,
-        }
-        llabel.orient_v = orient_v
-        llabel.render()
-        llabel.show()
+        # # left side label
```

Comment:
Yeah not convinced at all on these line labels minus the simple one for sizing. i'm thinkin we might want to swap out the contents labels with position and order info when order mode is engaged.